### DISCUSSION TITLE
Feature request: show the number of found results when searching

### DISCUSSION THREAD

**Discussion by Whiplashhh** [sumatrapdfreader/sumatrapdf#4451] [Ideas]
As title says, what I'd like to have is the info about number of found results in the whole document. It's also useful that it says which result is highlighted currently (for example: 4/98 - this tells me there's 98 results in total and that I'm currently on 4th of those). It doesn't feel like a hard thing to implement, but it's really useful to have it.

![image](https://github.com/user-attachments/assets/129ea191-5230-4a15-a38b-3aa9857bc246)

**Comment by kjk** [comment]
I opened https://github.com/sumatrapdfreader/sumatrapdf/issues/4452 to track this

**Comment by Whiplashhh** [comment]
Almost 5 months passed and such a simple feature isn't implemented yet. I hope it could be somehow prioritized. ;)

**Comment by Entony777** [comment]
Yes ! It wlll be nice feature !

PS: im using for this function - SIGIL, editor of EPUB files.
But i think implementing this to Sumatra will be nice.
